docs/docs/guides/integrate/login-ui/device-auth.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Support for the Device Authorization Grant in a Custom Login UI
+sidebar_label: Device Authorization
+---
+
+In case one of your applications requires the OAuth2 Device Authorization Grant this guide will show you how to implement
+this in your application as well as the custom login UI.
+
+The following flow shows you the different components you need to enable OAuth2 Device Authorization Grant for your login.
+![Device Auth Flow](/img/guides/login-ui/device-auth-flow.png)
+
+1. Your application makes a device authorization request to your login UI
+2. The login UI proxies the request to ZITADEL.
+3. ZITADEL parses the request and does what it needs to interpret certain parameters (e.g., organization scope, etc.)
+4. ZITADEL returns the device authorization response
+5. Your application presents the user_code and verification_uri or maybe even renders a QR code with the verification_uri_complete for the user to scan
+6. Your application starts a polling mechanism to check if the user has approved the device authorization request on the token endpoint
+7. When the user opens the browser at the verification_uri, he can enter the user_code, or it's automatically filled in, if they scan the QR code
+8. Request the device authorization request from the ZITADEL API using the user_code
+9. Your login UI allows to approve or deny the device request
+10. In case they approved, authenticate the user in your login UI by creating and updating a session with all the checks you need.
+11. Inform ZITADEL about the decision:
+    1. Authorize the device authorization request by sending the session and the previously retrieved id of the device authorization request to the ZITADEL API
+    2. In case they denied, deny the device authorization from the ZITADEL API using the previously retrieved id of the device authorization request
+12. Notify the user that they can close the window now and return to the application.
+13. Your applications request to the token endpoint now receives the tokens or an error if the user denied the request.
+
+## Example
+
+Let's assume you host your login UI on the following URL:
+```
+https://login.example.com
+```
+
+## Device Authorization Request
+
+A user opens your application and is unauthenticated, the application will create the following request:
+```HTTP
+POST /oauth/v2/device_authorization HTTP/1.1
+Host: login.example.com
+Content-type: application/x-www-form-urlencoded
+
+client_id=170086824411201793%40yourapp&
```

Comment:
Nitpicking, but we no longer issue `<client_id>@<project>` form client IDs anymore.

```suggestion
client_id=170086824411201793&
```